### DISCUSSION TITLE
Homework 2 Step 2 Feedback list released

### DISCUSSION THREAD

**Discussion by Ossifragus** [STAT5125-UConn/2022Spring#8] [Announcements]
The Feedback list is released ([d0946f3](https://github.com/STAT5125-UConn/2022Spring/commit/d0946f3b8866900518019044c5002d934dfaf5a2)), and it is due on Monday Feb 21. 

Note: If you did not submit your work before the due time or within the 24-hour grace period, your will not receive the credit and your name will not be on the  feedback list.

**Comment by andyslater** [comment]
Hi HaiYing,

 

I am verry sorry about not handing the HW in on time. I sent you an email on Friday.

I did not know we had 24hrs grace period… I did upload the assignment just tnow.

 

 

Regards, 

Andy Slater

 

 ***@***.***> ***@***.*** | 203 292 0002

 

 

 

From: HaiYing Wang ***@***.***> 
Sent: Saturday, February 19, 2022 8:52 PM
To: STAT5125-UConn/2022Spring ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [STAT5125-UConn/2022Spring] Homework 2 Step 2 Feedback list released (Discussion #8)

 

The Feedback list is released (d0946f3 <https://github.com/STAT5125-UConn/2022Spring/commit/d0946f3b8866900518019044c5002d934dfaf5a2> ), and it is due on Monday Feb 21.

Note: If you did not submit your work before the due time or within the 24-hour grace period, your will not receive the credit and your name will not be on the feedback list.

—
Reply to this email directly, view it on GitHub <https://github.com/STAT5125-UConn/2022Spring/discussions/8> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAM7GOSFA6DJEERRUUYGE23U4BCNLANCNFSM5O3XP4NA> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .

**Reply by Ossifragus** [reply]
The 24-hour grace period is written in the [syllabus](https://github.com/STAT5125-UConn/2022Spring/blob/main/Syllabus.org) we discussed in class. Be sure to submit future homework in time.

**Comment by andyslater** [comment]
Ok. Thanks for pointing this out.

 

 

Regards, 

Andy Slater

 

 ***@***.***> ***@***.*** | 203 292 0002
